tracer/time_windows.go

```diff
@@ -0,0 +1,31 @@
+package tracer
+
+import (
+	"golang.org/x/sys/windows"
+	"time"
+)
+
+// This method is more precise than the go1.8 time.Now on Windows
+// See https://msdn.microsoft.com/en-us/library/windows/desktop/hh706895(v=vs.85).aspx
+// It is however ~10x slower and requires Windows 8+.
```

Comment:
As a note, I suggested the same approach for a DNS Check in the Datadog agent: https://github.com/DataDog/integrations-core/blob/2209b0c316378fcde2563b49c9200e0ffef18d91/dns_check/check.py#L15-L26

Even if it's a different problem (a polling check that is processed in another process), it's still required because it gets wrong time on Windows.